pkg/client/load_balancer.go

```diff
@@ -0,0 +1,410 @@
+// (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	consts "github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/common"
+	"github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/models"
+)
+
+type LoadBalancerAPIService struct {
+	Client APIClientHandler
+	Cfg    Configuration
+}
+
+func (lb *LoadBalancerAPIService) CreateLoadBalancer(
+	ctx context.Context,
+	request models.CreateLoadBalancerRequest,
+) (models.CreateNetworkLoadBalancerResp, error) {
+	loadBalancerResp := models.CreateNetworkLoadBalancerResp{}
+	loadBalancerAPI := &api{
+		method: "POST",
+		path:   fmt.Sprintf("%s/%s", consts.BasePath, consts.LoadBalancerPath),
```

Comment:
Do we need consts.BasePath ?

Reply:
yes it will start with api/loadbalancer so BasePath is api

Reply:
This is addressed in another review. Resolving the conversation